helm/causelybot/templates/bot_secret.yaml

```diff
@@ -0,0 +1,20 @@
+{{- include "validateWebhookUrl" . -}}
+{{- include "validateBearerToken" . -}}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: slack-webhook-secret
+  namespace:  {{ .Release.Namespace }}
+type: Opaque
+data:
+  webhook-url: {{ .Values.webhook.url | b64enc }}
+
+---
+apiVersion: v1
```

Comment:
why do you need two secrets - these are used together

Reply:
Addressed in the new commit!